src/getESLint.js

```diff
@@ -5,10 +5,20 @@ const { Worker: JestWorker } = require('jest-worker');
 // @ts-ignore
 const { setup, lintFiles } = require('./worker');
 const { getESLintOptions } = require('./options');
-const { jsonStringifyReplacerSortKeys } = require('./utils');
 
-/** @type {{[key: string]: any}} */
-const cache = {};
+/** @type {Map<CacheKey, any>} */
+const cache = new Map();
```

Comment:
Is it possible to use WeakMap? Otherwise it is memory leak for incremental builds